bin/docs.test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -ev
+
+# Make sure we use the correct repo root dir
+DIR=`dirname $0`
+REPO_ROOT=$DIR/..
+pushd $REPO_ROOT >/dev/null
+rm -rf sandbox/loopback.io/
+# Shadow clone is faster
+git clone --depth 1 https://github.com/strongloop/loopback.io.git sandbox/loopback.io
+npm run bootstrap
+pushd $REPO_ROOT/sandbox/loopback.io/ >/dev/null
+# The following 3 lines will be replaced with npm run build
```

Comment:
Should we reword this comment as a TODO? Should this PR wait until `npm run build` is implemented in loopback.io?

Reply:
Already implemented. See https://github.com/strongloop/loopback.io/pull/666

Will merge it today, just wanted to make sure @raymondfeng was ok with excluding the `bundle install` from the `build` script.